packages/flutter/lib/src/material/banner.dart

```diff
@@ -86,25 +86,25 @@ class MaterialBanner extends StatefulWidget {
   /// The [actions], [content], and [forceActionsBelow] must be non-null.
   /// The [actions].length must be greater than 0. The [elevation] must be null or
   /// non-negative.
-  const MaterialBanner({
-    Key? key,
-    required this.content,
-    this.contentTextStyle,
-    required this.actions,
-    this.elevation,
-    this.leading,
-    this.backgroundColor,
-    this.padding,
-    this.leadingPadding,
-    this.forceActionsBelow = false,
-    this.overflowAlignment = OverflowBarAlignment.end,
-    this.animation,
-    this.onVisible
-  }) : assert(elevation == null || elevation >= 0.0),
-       assert(content != null),
-       assert(actions != null),
-       assert(forceActionsBelow != null),
-       super(key: key);
+  const MaterialBanner(
```

Comment:
Please do not reformat code (here and elsewhere)

Reply:
Okay